### PR TITLE
Add mongodb

### PR DESCRIPTION
**Pull request by abhijoshi2k**
## Checks

I have...

- [x] Read and understood the [Contributing Guidelines][contributing]
- [ ] Updated any nessecary files such as the README.md and/or CHANGELOG.md.

## Type of Pull request

- [ ] **Minor Change**  
  This Pull request doesn't break existing configuration.
- [x] **Major Change**  
  This Pull request will break existing configuration.
- [ ] **Bug fix**  
  This Pull request will fix a (critical) bug.
- [ ] **Documentation**  
  This Pull request only changes documentation (README.md, CHANGELOG.md, etc.)
- [ ] **Other:** `__________` 

## Description

Added mongodb and functions to update data, get data, create new data and delete data.
Note: These functions should be used with **async/await** as they are asynchronous functions

Closes #

[contributing]: https://github.com/Welcome-Bot/welcome-bot/blob/main/.github/CONTRIBUTING.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,19 +7,22 @@ Note that the displayed date is in the format `dd-mm-yyyy`
 [Older changelogs](#older-changelogs)
 
 ## [v0.2.0]
+
```
Comment (PuneetGopinath) [inline CHANGELOG.md:10]: Bro, even without these changes it will work

File: README.md [hunk 2]
```diff
@@ -9,33 +10,40 @@
...
+
 The bot requires the following permissions:
+
 - Read messages
 - Send messages
 - Read Message History
 
 To invite the bot with all the necessary required permissions mentioned above you can use this invitation link:
-> https://discord.com/oauth2/authorize?scope=bot&client_id=848459799783669790&permissions=68608
+> <https://discord.com/oauth2/authorize?scope=bot&client_id=848459799783669790&permissions=68608>
```
Comment (PuneetGopinath) [inline README.md:25]:
```suggestion
> https://discord.com/oauth2/authorize?scope=bot&client_id=848459799783669790&permissions=68608
```

File: README.md [hunk 3]
```diff
@@ -9,33 +10,40 @@
...
 
 If you want to add the bot without a `pre-made` role, use this link instead:
-> https://discord.com/oauth2/authorize?scope=bot&client_id=848459799783669790&permissions=0
+> <https://discord.com/oauth2/authorize?scope=bot&client_id=848459799783669790&permissions=0>
```
Comment (PuneetGopinath) [inline README.md:28]:
```suggestion
> https://discord.com/oauth2/authorize?scope=bot&client_id=848459799783669790&permissions=0
```

**Review by PuneetGopinath — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (PuneetGopinath) [reply CHANGELOG.md:10]: This is optional

File: README.md (hunk 2 above)
Comment (abhijoshi2k) [reply README.md:25]: ESlint shows this as violation. And this was the fix.

File: README.md (hunk 3 above)
Comment (abhijoshi2k) [reply README.md:28]: ESlint shows this as violation. And this was the fix.

File: README.md (hunk 2 above)
Comment (PuneetGopinath) [reply README.md:25]: Bro, say eslint to ignore readme files

Comment (PuneetGopinath) [reply README.md:25]: @abhijoshi2k

File: CHANGELOG.md (hunk 1 above)
Comment (abhijoshi2k) [reply CHANGELOG.md:10]: Yes it will work. But it sure does violate the rules.

**Review by PuneetGopinath — APPROVED** [review]

**Pull request merged**